src/main/java/io/zksync/wallet/DefaultZkSyncWallet.java

```diff
@@ -103,7 +101,7 @@ public String syncWithdraw(String ethAddress, BigInteger amount, TransactionFee
         final SignedTransaction<Withdraw> signedWithdraw = buildSignedWithdrawTx(ethAddress, fee.getFeeToken(), amount,
                 fee.getFee(), nonceToUse, timeRange);
 
-        return submitSignedTransaction(signedWithdraw.getTransaction(), signedWithdraw.getEthereumSignature(),
+        return submitSignedTransaction(signedWithdraw.getTransaction(), signedWithdraw.getEthereumSignature()[0],
```

Comment:
Why signed withdraw signature is an array?

Reply:
Some transaction types (like Swap) require multiple signatures. Thus in class SignedTransaction was added signature array instead of singe one